packages/@lwc/engine-server/src/__tests__/fixtures/light-dom/expected.html

```diff
@@ -1,8 +1 @@
-<x-basic>
-  <style type="text/css">
-    p {
-      color: red;
-    }
-  </style>
-  <p>Hello</p>
-</x-basic>
```

Comment:
Hm, another case where `engine-server` is affected... I may have to rethink this a bit for SSR.